cli/tools/jupyter/jupyter_msg.rs

```diff
@@ -15,6 +14,8 @@ use ring::hmac;
 use std::fmt;
 use uuid::Uuid;
 
+include!("../../util/time.rs");
```

Comment:
I think this will fail to cargo publish?

Reply:
I mean, below in ext/kv it will fail

Reply:
Agreed, maybe just duplicate it for now?

Reply:
Added duplicate in ext/kv